BackofficeBundle/Resources/public/coffee/orchestraLib.coffee

```diff
@@ -11,7 +11,7 @@ renderPageTitle = ->
 
 # ADD CUSTOM JARVIS WIDGET
 addCustomJarvisWidget = (widget) ->
-  $(widget).insertAfter($(".js-widget-title"))
+  $(widget).insertBefore($(".js-widget-title"))
```

Comment:
@itkg-ngilain I agree with you. This was the point that was bothering me the most in my own pr. What decided me was that the semantic result is identical to the header of modals. This change allowed only to maintain the buttons on the top right of the header.